nipype/interfaces/ants/registration.py

```diff
@@ -502,16 +502,32 @@ def _formatMetric(self, index):
             # dict-comprehension only works with python 2.7 and up
             #specs = [{k: v[i] for k, v in items} for i in indexes]
             specs = [dict([(k, v[i]) for k, v in items]) for i in indexes]
+            specs = list()
```

Comment:
@chrisfilo Line 504 should be deleted.  The simple logic of the dictionary comprehension needed to be expanded to the code between lines 505-518.  

Lines 502, 503, and 504 are no longer influencing the final values of specs, and the lines should have been removed because they are not used.

I will remove the old unused code.